pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -263,13 +263,23 @@ func (s *Server) loadCACerts(caOpts *caOptions, dir string) error {
 		return nil
 	}
 
-	signingKeyFile := path.Join(dir, ca.CAPrivateKeyFile)
-	if _, err := os.Stat(signingKeyFile); err == nil {
+	signingCABundleComplete, bundleExists, err := checkCABundleCompleteness(
```

Comment:
Add a comment here explaining that we first verify the CA isn't already volume mounted before attempting to load an external CA bundle